src/viewer/scene/CameraControl/CameraControl.js

```diff
@@ -921,6 +923,28 @@ class CameraControl extends Component {
         return this._configs.panRightClick;
     }
 
+    /**
+     * Sets whether mouse-moving after right-click rotates the {@link Camera}.
+     *
+     * Default is ````true````.
+     *
+     * @param {Boolean} value Set ````false```` to disable rotate on mouse-move after right-click.
+     */
+    set rotateRightClick(value) {
+        this._configs.rotateRightClick = value !== false;
```

Comment:
I think this should be false by default, so that left-drag-rotate is able to be default behaviour

Reply:
@xeolabs , the current behaviour is `true`. If we put `false` this will lead to minor breaking change... But if you think this is better I can update it.